### PR TITLE
Send møteinnkalling to Altinn when no nærmeste leder

### PR DESCRIPTION
**Pull request by kaja-nav**

### CODE REVIEW

**Comment by juneih** [comment]
Dette er ikke ferdig kode, men endringen begynner å bli stor så vi ønsker å dele den opp. Koden her er i praksis togglet, fordi vi ikke har skrudd på i frontend for sykmeldte uten nærmeste leder.

File: build.gradle.kts [hunk 1]
```diff
@@ -23,14 +25,18 @@ object Versions {
     const val logstashEncoder = "7.0.1"
     const val mockk = "1.12.2"
     const val nimbusjosejwt = "9.18"
-    const val postgresEmbedded = "0.13.4"
+    const val postgresEmbedded = "1.0.0"
```
Comment (andersrognstad) [inline build.gradle.kts:28]: Litt usikker på om vi kan gjøre dette. Det ble forsøkt å oppgradere denne tidligere, men da fikk man problemer med å kjøre tester lokalt (på Windows).

Comment (geir-waagboe) [reply build.gradle.kts:28]: Ja, best om dette reverseres.

File: src/main/kotlin/no/nav/syfo/client/altinn/AltinnMapper.kt [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+import no.altinn.schemas.services.serviceengine.subscription._2009._10.AttachmentFunctionType
+import no.altinn.services.serviceengine.reporteeelementlist._2010._10.BinaryAttachmentExternalBEV2List
+import no.altinn.services.serviceengine.reporteeelementlist._2010._10.BinaryAttachmentV2
+import no.nav.syfo.domain.Virksomhetsnummer
+import java.lang.Boolean.FALSE
+import java.util.*
+
+class AltinnMapper private constructor() {
+    companion object {
+        const val DIALOGMOTE_TJENESTEKODE =
```
Comment (Vitenok) [inline src/main/kotlin/no/nav/syfo/client/altinn/AltinnMapper.kt:16]: Bør `DIALOGMOTE_TJENESTEKODE` være privat? Ser ikke at den blir brukt andre steder

File: build.gradle.kts [hunk 3]
```diff
@@ -1,4 +1,6 @@
+import Build_gradle.Versions.altinnCorrespondenceAgencyExternalVersion
```
Comment (geir-waagboe) [inline build.gradle.kts:1]: Ser litt rart ut å ha en import på denne. Burde ikke være nødvendig siden det er i samme fila som den er definert.

File: src/main/kotlin/no/nav/syfo/application/api/ApiModule.kt [hunk 4]
```diff
@@ -139,6 +155,7 @@ fun Application.apiModule(
         oppfolgingstilfelleClient = oppfolgingstilfelleClient,
         pdfGenClient = pdfGenClient,
         kontaktinformasjonClient = kontaktinformasjonClient,
+        varselService = varselService,
```
Comment (andersrognstad) [inline src/main/kotlin/no/nav/syfo/application/api/ApiModule.kt:158]: Fint med egen varselService 👍 Ser ut til at man kan fjerne de tre andre varselservicene fra konstruktøren til DialogmoteService nå.

File: src/main/kotlin/no/nav/syfo/client/altinn/AltinnClient.kt [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+    private fun sendMelding(virksomhetsBrevAltinnWS: InsertCorrespondenceV2, brevId: UUID): Int {
+        try {
+            val receiptWS = iCorrespondenceAgencyExternalBasic.insertCorrespondenceBasicV2(
+                username,
+                password,
+                SYSTEM_USER_CODE,
+                brevId.toString(),
+                virksomhetsBrevAltinnWS
+            )
+            if (receiptWS.receiptStatusCode != ReceiptStatusEnum.OK) {
```
Comment (andersrognstad) [inline src/main/kotlin/no/nav/syfo/client/altinn/AltinnClient.kt:28]: Her kommer det kanskje noen metrikker senere (antal success/fail e.l.)?

File: src/main/kotlin/no/nav/syfo/client/altinn/AltinnClient.kt [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+                throw RuntimeException("Error from altinn")
+            }
+            return receiptWS.receiptId
+        } catch (ex: Exception) {
+            log.error("Error sending brev to altinn", ex)
+            throw ex
+        }
+    }
+
+    fun sendToVirksomhet(
```
Comment (andersrognstad) [inline src/main/kotlin/no/nav/syfo/client/altinn/AltinnClient.kt:44]: Fint om public-metode kommer før private-metode i fila her

File: src/main/kotlin/no/nav/syfo/dialogmote/VarselService.kt [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+                createdAt = tidspunktForVarsel,
+                personIdent = arbeidstakerPersonIdent,
+                type = varselType,
+                motedeltakerArbeidstakerUuid = arbeidstakerId,
+                varselUuid = arbeidstakerbrevId,
+            )
+        }
+
+        // TODO Send varsel til leder for innkalling, endring og avlysning
+        if (narmesteLeder != null) {
```
Comment (Vitenok) [inline src/main/kotlin/no/nav/syfo/dialogmote/VarselService.kt:53]: Erdet glemt `TODO`?

File: src/main/kotlin/no/nav/syfo/dialogmote/VarselService.kt [hunk 8]
```diff
@@ -0,0 +1,77 @@
...
+            narmesteLederVarselService.sendVarsel(
+                createdAt = tidspunktForVarsel,
+                moteTidspunkt = moteTidspunkt,
+                narmesteLeder = narmesteLeder,
+                varseltype = varselType
+            )
+        } else {
+            // TODO Gjør dette først
```
Comment (Vitenok) [inline src/main/kotlin/no/nav/syfo/dialogmote/VarselService.kt:61]: Erdet glemt `TODO`?

File: src/main/kotlin/no/nav/syfo/client/altinn/AltinnClient.kt [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+import no.nav.syfo.domain.Virksomhetsnummer
+import org.slf4j.LoggerFactory
+import java.util.*
+
+class AltinnClient(
+    private val iCorrespondenceAgencyExternalBasic: ICorrespondenceAgencyExternalBasic,
+    private val username: String,
+    private val password: String
+) {
+    val SYSTEM_USER_CODE = "NAV_DIGISYFO"
```
Comment (andersrognstad) [inline src/main/kotlin/no/nav/syfo/client/altinn/AltinnClient.kt:16]: Denne kan være private

File: src/main/kotlin/no/nav/syfo/dialogmote/VarselService.kt (hunk 7 above)
Comment (JMLindseth) [reply src/main/kotlin/no/nav/syfo/dialogmote/VarselService.kt:53]: Denne tror jeg bare kan fjernes 👍

File: src/main/kotlin/no/nav/syfo/dialogmote/VarselService.kt (hunk 8 above)
Comment (JMLindseth) [reply src/main/kotlin/no/nav/syfo/dialogmote/VarselService.kt:61]: Dette tenkte vi å gjøre etter hvert 🙂 Tanken er å flytte altinn-biten først, så vi ikke prøver å gjøre noe annet hvis det kallet feiler.

File: src/test/kotlin/no/nav/syfo/dialogmote/api/v2/PostDialogmoteApiV2Spek.kt [hunk 10]
```diff
@@ -337,6 +337,8 @@ class PostDialogmoteApiV2Spek : Spek({
                         }
                     }
 
+                    // TODO Endre denne til at vi sender til altinn når leder mangler, og kanskje tilsvarende i de andre testene?
```
Comment (Vitenok) [inline src/test/kotlin/no/nav/syfo/dialogmote/api/v2/PostDialogmoteApiV2Spek.kt:340]: Er det fortsatt WIP?

File: src/main/kotlin/no/nav/syfo/application/api/ApiModule.kt [hunk 11]
```diff
@@ -98,6 +100,13 @@ fun Application.apiModule(
         syfotilgangskontrollClientId = environment.syfotilgangskontrollClientId,
         tilgangskontrollBaseUrl = environment.syfotilgangskontrollUrl
     )
+
+    val altinnClient = AltinnClient(
+        iCorrespondenceAgencyExternalBasic = createPort(environment.altinnWsUrl),
```
Comment (andersrognstad) [inline src/main/kotlin/no/nav/syfo/application/api/ApiModule.kt:105]: Kanskje bedre om AltinnClient tar seg av å lage en sånn `ICorrespondenceAgencyExternalBasic`-instans? Dvs bare sende inn environment-variablene til AltinnClient så fikser den Altinn-spesifikke greier.

File: src/test/kotlin/no/nav/syfo/dialogmote/api/v2/PostDialogmoteApiV2Spek.kt [hunk 12]
```diff
@@ -337,6 +337,8 @@ class PostDialogmoteApiV2Spek : Spek({
...
+                    // VI trenger vel også en mockserver for altinn?
```
Comment (Vitenok) [inline src/test/kotlin/no/nav/syfo/dialogmote/api/v2/PostDialogmoteApiV2Spek.kt:341]: Er det fortsatt WIP? Bør det kanskje være trello-lapp i stedet?

File: src/main/kotlin/no/nav/syfo/client/altinn/AltinnClient.kt [hunk 13]
```diff
@@ -0,0 +1,52 @@
...
+    private val log = LoggerFactory.getLogger(BehandlerDialogmeldingProducer::class.java)
```
Comment (andersrognstad) [inline src/main/kotlin/no/nav/syfo/client/altinn/AltinnClient.kt:17]: Skulle man brukt AltinnClient som klasse-parameter her?

File: src/main/kotlin/no/nav/syfo/client/altinn/AltinnClient.kt (hunk 9 above)
Comment (Vitenok) [inline src/main/kotlin/no/nav/syfo/client/altinn/AltinnClient.kt:16]: Kan være private kanskje?

File: src/test/kotlin/no/nav/syfo/dialogmote/api/v2/PostDialogmoteApiV2Spek.kt [hunk 14]
```diff
@@ -337,6 +337,8 @@ class PostDialogmoteApiV2Spek : Spek({
...
                     it("should return status InternalServerError if denied person with Dialogmote with Virksomhet does not have a leader for that Virksomhet") {
```
Comment (andersrognstad) [inline src/test/kotlin/no/nav/syfo/dialogmote/api/v2/PostDialogmoteApiV2Spek.kt:342]: Burde ikke denne testen feilet nå? Siden det ble fjernet sjekk på `narmesteLeder == null`

File: build.gradle.kts (hunk 1 above)
Comment (juneih) [reply build.gradle.kts:28]: Jeg fikk det også til å kjøre i IntelliJ

Comment (JMLindseth) [reply build.gradle.kts:28]: Jeg fikk det også til å kjøre i IntelliJ

File: src/main/kotlin/no/nav/syfo/client/altinn/AltinnClient.kt (hunk 5 above)
Comment (JMLindseth) [reply src/main/kotlin/no/nav/syfo/client/altinn/AltinnClient.kt:28]: Vi skriver det på lista!

File: src/test/kotlin/no/nav/syfo/dialogmote/api/v2/PostDialogmoteApiV2Spek.kt (hunk 14 above)
Comment (JMLindseth) [reply src/test/kotlin/no/nav/syfo/dialogmote/api/v2/PostDialogmoteApiV2Spek.kt:342]: Nå blir testen grønn, fordi kallet til altinn feiler, og da ender man oppe med 500-feil, men av feil årsak.

File: src/test/kotlin/no/nav/syfo/dialogmote/api/v2/PostDialogmoteApiV2Spek.kt (hunk 12 above)
Comment (JMLindseth) [reply src/test/kotlin/no/nav/syfo/dialogmote/api/v2/PostDialogmoteApiV2Spek.kt:341]: Vi har det på todoen vår, så det blir gjort i neste omgang 👍

File: src/test/kotlin/no/nav/syfo/dialogmote/api/v2/PostDialogmoteApiV2Spek.kt (hunk 14 above)
Comment (andersrognstad) [reply src/test/kotlin/no/nav/syfo/dialogmote/api/v2/PostDialogmoteApiV2Spek.kt:343]: Ja, nettopp, skjønner. Husk å fikse TODO/mockserver senere da 👍

File: src/main/kotlin/no/nav/syfo/client/altinn/AltinnMapper.kt [hunk 15]
```diff
@@ -0,0 +1,77 @@
...
+                .withAllowForwarding(FALSE)
+                .withReportee(virksomhetsnummer.value)
+                .withMessageSender(AVSENDER_NAV)
+                .withServiceCode(DIALOGMOTE_TJENESTEKODE)
+                .withServiceEdition(DIALOGMOTE_TJENESTEVERSJON)
+                .withNotifications(NotificiationAltinnGenerator.createNotifications())
+                .withContent(
+                    ExternalContentV2()
+                        .withLanguageCode(NORSK_BOKMAL)
+                        .withMessageTitle("Dialogmøte hos NAV")
```
[the remark this answers is not in the document]
Comment (juneih) [reply src/main/kotlin/no/nav/syfo/client/altinn/AltinnMapper.kt:41]: Nå behandler vi alle typer brev likt, men det ble ikke bra nok i Altinn så vi skal legge på skille slik at det blir fskj titler, filnavn osv for innkalling, endring ++. Vi tar med denne tilbakemeldingen til når vi endrer det :).

File: src/test/kotlin/no/nav/syfo/testhelper/UserConstants.kt [hunk 16]
```diff
@@ -31,6 +31,7 @@ object UserConstants {
 
     val NARMESTELEDER_FNR = PersonIdentNumber("98765432101")
     val NARMESTELEDER_FNR_2 = PersonIdentNumber("98765432102")
+    val VIRKSOMHETSNUMMER = Virksomhetsnummer("911111111")
```
Comment (janzh) [inline src/test/kotlin/no/nav/syfo/testhelper/UserConstants.kt:34]: Endre til `VIRKSOMHETSNUMMER_NO_NARMESTELEDER`?

File: src/main/kotlin/no/nav/syfo/client/altinn/AltinnClient.kt [hunk 17]
```diff
@@ -0,0 +1,51 @@
...
+    altinnWsUrl: String,
+    private val username: String,
+    private val password: String
+) {
+    private val SYSTEM_USER_CODE = "NAV_DIGISYFO"
+    private val log = LoggerFactory.getLogger(AltinnClient::class.java)
+    private val iCorrespondenceAgencyExternalBasic = createPort(altinnWsUrl)
+
+    fun sendToVirksomhet(
+        brevId: UUID,
```
Comment (janzh) [inline src/main/kotlin/no/nav/syfo/client/altinn/AltinnClient.kt:19]: brevId -> brevUUid?

File: src/main/kotlin/no/nav/syfo/client/altinn/AltinnClient.kt [hunk 18]
```diff
@@ -0,0 +1,51 @@
...
+        brevPdf: ByteArray,
+        virksomhetsnummer: Virksomhetsnummer,
+    ) {
+        val virksomhetsBrevAltinnWS = AltinnMapper.brevTilWSCorrespondence(brevId, brevPdf, virksomhetsnummer)
```
Comment (janzh) [inline src/main/kotlin/no/nav/syfo/client/altinn/AltinnClient.kt:23]: Bruke named parameters her?

File: src/main/kotlin/no/nav/syfo/client/altinn/AltinnClient.kt [hunk 19]
```diff
@@ -0,0 +1,51 @@
...
+        sendMelding(virksomhetsBrevAltinnWS, brevId)
```
Comment (janzh) [inline src/main/kotlin/no/nav/syfo/client/altinn/AltinnClient.kt:24]: Bruke named parameters i disse to funksjonskallene?

File: build.gradle.kts (hunk 1 above)
Comment (geir-waagboe) [reply build.gradle.kts:27]: Funker dessverre *ikke* med 1.0.0 når jeg kjører ./gradlew clean build fra kommandolinja.

File: src/main/kotlin/no/nav/syfo/dialogmote/DialogmoteService.kt [hunk 20]
```diff
@@ -111,98 +114,95 @@ class DialogmoteService(
...
-                callId = callId,
-                personIdentNumber = personIdentNumber,
-                token = token,
-            ) ?: throw RuntimeException("Failed to request BehandlendeEnhet of Person")
 
-            val newDialogmote = newDialogmoteDTO.toNewDialogmote(
-                requestByNAVIdent = getNAVIdentFromToken(token),
-                navEnhet = EnhetNr(behandlendeEnhet.enhetId),
-            )
+        val behandlendeEnhet = behandlendeEnhetClient.getEnhet(
```
[the remark this answers is not in the document]
Comment (geir-waagboe) [reply src/main/kotlin/no/nav/syfo/dialogmote/DialogmoteService.kt:115]: Mener du at det er Modia som "toggler" dette? Jeg synes nok at det hadde vært bra med en toggle i isdialogmote i tillegg.

File: src/test/kotlin/no/nav/syfo/dialogmote/VarselServiceSpek.kt [hunk 21]
```diff
@@ -0,0 +1,229 @@
...
+import no.nav.syfo.testhelper.UserConstants
+import no.nav.syfo.testhelper.UserConstants.VIRKSOMHETSNUMMER
+import no.nav.syfo.testhelper.UserConstants.VIRKSOMHETSNUMMER_HAS_NARMESTELEDER
+import no.nav.syfo.testhelper.mock.narmesteLeder
+import org.spekframework.spek2.Spek
+import org.spekframework.spek2.style.specification.describe
+import java.time.LocalDateTime
+import java.util.*
+
+class VarselServiceSpek : Spek({
```
[the remark this answers is not in the document]
Comment (geir-waagboe) [reply src/test/kotlin/no/nav/syfo/dialogmote/VarselServiceSpek.kt:21]: Jeg synes ikke det er noe i veien for at det er en egen enhetstest for denne komponenten (i tillegg til de andre integrasjonstestene).

File: build.gradle.kts (hunk 1 above)
Comment (janzh) [reply build.gradle.kts:27]: Stemmer det @geir-waagboe sier og hos meg fungerer det ikke IntelliJ heller. Jeg får følgende feilmelding:
"Previous attempts to find a Docker environment failed. Will not retry. Please see logs and check configuration".

Når dere sier at det fungerer i Intellij hos dere på windows, er dere sikre på at dere har oppdatert depedency og at dere ikke har liggende v. 0.13.4 når dere kjøre test? Om jeg skal tippe har dere ikke oppdatert og hentet inn ny versjon.

I v. 1.0.0 så har de lagt til noe nye greier som krever et docker-env og ser ikke at dette fungerer automagisk på windows.

File: src/test/kotlin/no/nav/syfo/dialogmote/VarselServiceSpek.kt (hunk 21 above)
[the remark this answers is not in the document]
Comment (janzh) [reply src/test/kotlin/no/nav/syfo/dialogmote/VarselServiceSpek.kt:21]: Er jo ikke krise, men ser ikke helt poenget med å teste ting dobbelt.

[the remark this answers is not in the document]
Comment (andersrognstad) [reply src/test/kotlin/no/nav/syfo/dialogmote/VarselServiceSpek.kt:21]: Jeg tenker også det kan være fint med egne unit-tester på services hvis det er logikk og edge-cases der man ønsker å teste isolert. Samtidig som man f.eks. tester mer happy case i integrasjonstestene.

[the remark this answers is not in the document]
Comment (janzh) [reply src/test/kotlin/no/nav/syfo/dialogmote/VarselServiceSpek.kt:21]: Ja, men hva er det denne testen dekker som ikke allerede gjøres i eksisterende integrasjonstester?
Jeg kan ikke se at denne testen gjør spesielt eller tester edge-caser, såvidt jeg kan se er det en stor fil med kode og som tester lite.

Såvidt jeg vet har vi allerede tester for arbeidstaker som er reservert og for arbeidstaker som ikke er reservert. Er det ikke bare å legge på verify(sendToAltinn) der hvor det trengs i eksisterende filer?

[the remark this answers is not in the document]
Comment (juneih) [reply src/test/kotlin/no/nav/syfo/dialogmote/VarselServiceSpek.kt:21]: Ah jeg skjønner. Vi sender ikke alltid til altinn, kun hvis nærmeste leder ikke er registrert.

[the remark this answers is not in the document]
Comment (janzh) [reply src/test/kotlin/no/nav/syfo/dialogmote/VarselServiceSpek.kt:21]: Ta f.eks et søk i Isdialogmote med følgende streng: `verify(exactly = ` så ser det ut som det meste er dekket og man kan legge til en altinn-verify der det trengs.

File: src/main/kotlin/no/nav/syfo/client/altinn/AltinnMapper.kt [hunk 22]
```diff
@@ -0,0 +1,77 @@
...
+import no.altinn.schemas.services.serviceengine.correspondence._2010._10.UserTypeRestriction
+import no.altinn.schemas.services.serviceengine.subscription._2009._10.AttachmentFunctionType
+import no.altinn.services.serviceengine.reporteeelementlist._2010._10.BinaryAttachmentExternalBEV2List
+import no.altinn.services.serviceengine.reporteeelementlist._2010._10.BinaryAttachmentV2
+import no.nav.syfo.domain.Virksomhetsnummer
+import java.lang.Boolean.FALSE
+import java.util.*
+
+class AltinnMapper private constructor() {
+    companion object {
```
[the remark this answers is not in the document]
Comment (juneih) [reply src/main/kotlin/no/nav/syfo/client/altinn/AltinnMapper.kt:15]: Det er kopiert fra Team Sykmelding går ut fra det er for å kunne kalle funksjonen direkte.

File: src/test/kotlin/no/nav/syfo/dialogmote/VarselServiceSpek.kt (hunk 21 above)
[the remark this answers is not in the document]
Comment (juneih) [reply src/test/kotlin/no/nav/syfo/dialogmote/VarselServiceSpek.kt:21]: Fjernet de unittestene som var dekket av andre integrasjonstester. Resten av testen forsvinner antagelig helt når alt skal til altinn.

**Review by torhrise — APPROVED** [review]

File: src/main/kotlin/no/nav/syfo/client/altinn/AltinnClient.kt (hunk 19 above)
Comment (JMLindseth) [reply src/main/kotlin/no/nav/syfo/client/altinn/AltinnClient.kt]: funksjon💀💀

File: src/main/kotlin/no/nav/syfo/client/altinn/AltinnMapper.kt (hunk 22 above)
[the remark this answers is not in the document]
Comment (cyanboy) [reply src/main/kotlin/no/nav/syfo/client/altinn/AltinnMapper.kt:15]: Hvis de bare skal kalles direkte, er det nødvendig med en klasse da? Kan ikke bare funksjoner og konstanter flyttes ut?

File: src/main/kotlin/no/nav/syfo/client/altinn/NotificiationAltinnGenerator.kt [hunk 23]
```diff
@@ -0,0 +1,64 @@
+package no.nav.syfo.client.altinn
+
+import no.altinn.schemas.serviceengine.formsengine._2009._10.TransportType
+import no.altinn.schemas.services.serviceengine.notification._2009._10.*
+
+class NotificiationAltinnGenerator {
```
Comment (cyanboy) [inline src/main/kotlin/no/nav/syfo/client/altinn/NotificiationAltinnGenerator.kt:6]: Samme med denne, trenger dette egentlig være en klasse?

File: src/test/kotlin/no/nav/syfo/dialogmote/VarselServiceSpek.kt [hunk 24]
```diff
@@ -0,0 +1,113 @@
...
+import no.nav.syfo.testhelper.UserConstants
+import no.nav.syfo.testhelper.UserConstants.VIRKSOMHETSNUMMER_NO_NARMESTELEDER
+import no.nav.syfo.testhelper.UserConstants.VIRKSOMHETSNUMMER_HAS_NARMESTELEDER
+import no.nav.syfo.testhelper.mock.narmesteLeder
+import org.spekframework.spek2.Spek
+import org.spekframework.spek2.style.specification.describe
+import java.time.LocalDateTime
+import java.util.*
+
+class VarselServiceSpek : Spek({
```
Comment (cyanboy) [inline src/test/kotlin/no/nav/syfo/dialogmote/VarselServiceSpek.kt:21]: Her er testen deklarert 
```kotlin
class VarselServiceSpek : Spek
```

men i `AltinnMapperSpek.kt` er testen deklarert 
```kotlin
object AltinnMapperSpek : Spek
```

Er det noen grunn til dette? I Spek-dokumentasjonen ser det ut som de foretrekker `object` måten.

File: src/main/kotlin/no/nav/syfo/client/altinn/AltinnMapper.kt (hunk 22 above)
[the remark this answers is not in the document]
Comment (geir-waagboe) [reply src/main/kotlin/no/nav/syfo/client/altinn/AltinnMapper.kt:15]: Enig i at det antakelig er bedre å kutte ut klassen her og forenkle.

File: src/main/kotlin/no/nav/syfo/client/altinn/NotificiationAltinnGenerator.kt (hunk 23 above)
Comment (geir-waagboe) [reply src/main/kotlin/no/nav/syfo/client/altinn/NotificiationAltinnGenerator.kt:6]: Enig.

File: src/test/kotlin/no/nav/syfo/dialogmote/VarselServiceSpek.kt (hunk 24 above)
Comment (juneih) [reply src/test/kotlin/no/nav/syfo/dialogmote/VarselServiceSpek.kt:21]: Startet også som kopi fra smaltinn, jeg har ingen sterke meninger om dette :). Bytter til object

**Review by cyanboy — APPROVED** [review]
